crmsh/cibconfig.py

```diff
@@ -2890,6 +2890,32 @@ def prim_id_list(self):
         "List of primitives ids (for group completion)."
         return [x.obj_id for x in self.cib_objects if x.obj_type == "primitive"]
 
+    def fence_id_list(self):
+        """
+        List all configured fence agent's id
+        """
+        return [x.obj_id for x in self.cib_objects if x.node.get('class') == "stonith"]
+
+    def fence_id_list_with_pcmk_delay(self):
```

Comment:
1. I wonder to be more strict criteria of this function is to check `pcmk_delay_max != 0`. In case some left over CIB with explicit `pcmk_delay_max=0`, in theory.

2. I have no obvious use case to include  `pcmk_delay_base` here. Along with this, I would prefer the precise function naming: fence_id_list_with_pcmk_delay_max()

Reply:
> 1. I wonder to be more strict criteria of this function is to check `pcmk_delay_max != 0`. In case some left over CIB with explicit `pcmk_delay_max=0`, in theory.

changed when the `value > 0`

> 2. I have no obvious use case to include  `pcmk_delay_base` here. Along with this, I would prefer the precise function naming: fence_id_list_with_pcmk_delay_max()

I prefer to keep this since I see it's the same purpose with pcmk_delay_max. OK, no harm to keep it

Reply:
I insist to remove the part in related to pcmk_delay_base here. Reasons are:
a) no use case deals with it so far. 
b) using it here just creates unnecessary brain power to understand the code, and to think about why, in the end it is not related to pcmk_delay_max use case at all.

Reply:
Already changed. removed the base one